scripts/build/BUILD.gn

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("//build_overrides/pigweed.gni")
+import("$dir_pw_build/python.gni")
+
+import("${chip_root}/build/chip/tests.gni")
+
+pw_python_package("build_examples") {
+  setup = [ "setup.py" ]
+  sources = [
+    "build/__init__.py",
+    "build/factory.py",
+    "build/targets.py",
+    "build_examples.py",
+    "builders/__init__.py",
+    "builders/builder.py",
+    "builders/efr32.py",
+    "builders/esp32.py",
+    "builders/linux.py",
+    "builders/qpg.py",
+    "expected_all_platform_commands.txt",
```

Comment:
Agree this is  wrong. We should find a better fix (if necessary plumb test data dependencies through pw_python_package).

Reply:
any suggestions?

Approaches I can think of:
   - leave alone as is with a comment of 'this is bad' 
   - place it inside a multiline string in test.py. I find placing text in python a bit ugly
   - create a python action to generate test.py out of a template test.py and the txt file, then the action can take these as inputs. I find this a bit overkilll and likely more error prone (the generated file moves into the build output which will create path headaches for me)
   
 I would go with the comment first, otherwise with multiline. However I find all approaches not ideal. I would rather have a 'test data deps' property in GN :/

Reply:
I added a comment for now.

Reply:
Asked pigweed folks, got recommendation to use 'inputs' instead of sources for this. Did so and it seems to work.